### DISCUSSION TITLE
网站接入openAI translator

### DISCUSSION THREAD

**Discussion by xiongvalerio** [nextai-translator/nextai-translator#1550] [General]
能否实现 自己的网站在代码层接入openAI translator 实现进入网站自动翻译的效果（翻译内容代替原有网站内容）.

**Comment by Xiaotian-Liu-MKT** [comment]
有个叫“沉浸式翻译”的插件感觉会符合你的需求。
<img width="213" alt="image" src="https://github.com/openai-translator/openai-translator/assets/49262952/13cffce4-8a0d-46fd-b813-ef553326addd">
